### DISCUSSION TITLE
Getting Error tableName not defined when exporting to CSV or JSON

### DISCUSSION THREAD

**Discussion by mintonan** [Budibase/budibase#9867] [Help] [answered]
So I am trying to test exporting data to a CSV, I am using MSSQL server and trying to export data from a table using selected rows. 
![image](https://user-images.githubusercontent.com/124909937/222564068-0d7707f8-54f6-431f-9326-94c27c5ecf2c.png)

I have made sure that my table name is defined correctly and I am choosing the correct table (There is only one option anyway on this screen) 
![image](https://user-images.githubusercontent.com/124909937/222565424-332d0962-644b-433c-ba87-cd033547a7fc.png)

I checked into the console and saw that I received this error:
`{"message":"RequestError: Conversion failed when converting the nvarchar value 'exportRows' to data type int.","status":500}`

Not sure where else to check as I even followed the video tutorial on actions step for step and am still running into this weird error

**Comment by melohagan** [comment, marked as answer]
Hey @mintonan 

What version of Budibase are you on?

**Reply by mintonan** [reply]
Hi @melohagan!

I am currently on 2.3.5 right now

**Reply by melohagan** [reply]
Could you try updating to the latest version please? Should be **2.3.16**

**Reply by mintonan** [reply]
Yep, looks like that worked! Was able to download a CSV from my selected rows. Thanks!
